scripts/check_ecosystem.py

```diff
@@ -111,6 +125,11 @@ async def check(
         ruff_args.extend(["--ignore", ignore])
     if exclude:
         ruff_args.extend(["--exclude", exclude])
+    if show_fixes:
+        # Generating fixes is slow and verbose
```

Comment:
Nit: Maybe move the comment to `Repository.show_fixes` because this information is useful when deciding whether to enable fixes or not for a specific repository